### ISSUE TITLE
Evaluate if/else conditionals

### ISSUE THREAD

**Issue by nurugger07** [nurugger07/calliope#20]
Evaluating `if/else` conditionals requires compiling a smart script.
Given the following haml:
``` haml
- if true do
  %p foo
- else
  %p bar
```
Then is should output the following html:
``` html
<p>foo</p>
```

**Comment by smpallen99** [comment]
Give that the EEx syntax for this is:
```
<%= if true do %>
  <p>foo</p>
<% else %>
  <p>bar</p>
<% end %>
```
Would it not be more appropriate to use the following:
```
= if true do
  %p foo
- else
  %p bar
```

**Comment by smpallen99** [comment]
I just figured out the difference between haml's silent script `-` and calliope's smart script. perhaps I'm my above suggestion is not the correct way to go. I'll withdraw the above suggestion.

**Comment by nurugger07** [comment]
Closed with [PR #40](https://github.com/nurugger07/calliope/pull/40)
